src/main/java/com/influxdb/v3/client/internal/RestClient.java

```diff
@@ -140,7 +141,29 @@ public void checkServerTrusted(
         this.client = builder.build();
     }
 
-    void request(@Nonnull final String path,
+    public String getServerVersion() throws InfluxDBApiException {
+        HttpResponse<String> response;
+        String influxdbVersion;
+        try {
+            response = request("ping", HttpMethod.GET, null, null, null);
+            influxdbVersion = extractServerVersion(response);
+        } catch (Exception e) {
+            throw new InfluxDBApiException(e);
+        }
+
+        return influxdbVersion;
```

Comment:
no need to re-wrap exceptions from `request()`
```suggestion
        HttpResponse<String> response = request("ping", HttpMethod.GET, null, null, null);
        try {
            return extractServerVersion(response);
        } catch (Exception e) {
            throw new InfluxDBApiException(e);
        }
```